app/components/login_button_component.scss

```diff
@@ -0,0 +1,43 @@
+@use 'uswds-core' as *;
+
+.login-button.usa-button--big > .login-button-img {
+    width: 10rem;
+}
+
+.login-button > .login-button-img {
+    width: 8rem;
+}
+
+.login-button.login-button--primary-lighter {
+    @include u-bg('primary-lighter');
```

Comment:
do we typically indent by 2 in CSS like we do for our other files? (.js, .rb)

Reply:
surprised the linter didn't fix it - yes I will fix!

Reply:
Looks like our linter isn't checking these component stylesheets. I can help put together a fix for that

https://github.com/18F/identity-idp/blob/dd963f150054e83f690a18c2fa57e43d7df7b11c/package.json#L14